src/config.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Module for getting and setting Prebid configuration.
+ *
+ * Prebid previously defined these properties directly on the global object:
+ * pbjs.logging = true;
+ *
+ * Defining and access properties in this way is now deprecated, but these will
+ * continue to work during a deprecation window.
+ */
+const utils = require('./utils');
+
+const DEFAULT_DEBUG = false;
+const DEFAULT_BIDDER_TIMEOUT = 3000;
+const DEFAULT_PUBLISHER_DOMAIN = window.location.origin;
+const DEFAULT_COOKIESYNC_DELAY = 100;
+const DEFAULT_ENABLE_SEND_ALL_BIDS = false;
+
+const ALL_TOPICS = '*';
+
+let listeners = [];
```

Comment:
Can you describe what you're thinking?  I think what we have here is pretty good.  

I don't really consider this global state as its encapsulated to the config module; however, it is "side-effecty", but I'm not sure that's a terrible thing.  However, it does need a way to clean up its side-effects for testing purposes, i.e. we need some way to remove listeners.